test/mock/plugins-lambda/src/plugins/custom-pubsub.js

```diff
@@ -0,0 +1,15 @@
+let { join } = require('path')
+
+module.exports = {
+  pluginFunctions: function (arc, inventory) {
+    let cwd = inventory.inv._project.src
+    return inventory.inv._project.arc.pubsub.map((channel) => {
+      return {
+        src: join(cwd, 'src', 'pubsub', channel),
+        name: channel,
+        body: `exports.handler = async function (event) { console.lo
```

Comment:
`body` is a property unique to this test – Inventory has largely opted not to deal with userland file contents (outside config, ofc), and let the consumers of Inventory load what they need from `src`, `handler`, etc. Any thoughts on that?

Reply:
I based this test on my prototype plugin implementation - so from the perspective of the plugin author. You are absolutely correct that Inventory does not deal with `body`, and therefore that property likely doesn't belong in the test. However, a plugin author will need to have their `pluginFunctions` return objects with a `body` property as this will get passed to `@architect/create` in order to create the file in the right place.

I will remove it from the test as its irrelevant in this context.

Reply:
I'll remove it, though if someone were to use this mock plugin as an example of one, it would technically not work (since `create` uses `body` to populate the template contents of the plugin lambda's handler file).